src/index.js

```diff
@@ -235,7 +239,7 @@ function displaySelectedClue(event) {
   selectedClue = clueInfo.find(clue => clue.id == clickedCard.id)
   clueCards.classList.add('no-clicks');
   turns ++;
-  if (turns === randomNumber1 || randomNumber2 || randomNumber3) {
+  if (turns === randomNumber1 || turns === randomNumber2 || turns === randomNumber3) {
```

Comment:
This is interesting -- are you creating the daily doubles based on turn number and not clue position?

Reply:
Yes, it's based on the turns. When we began this functionality, we never made an array of all total clues, so at the start of each turn we increment the turn count and if that matches a random number, it ends up being a daily double.